src/runtime/ArrayBufferObject.cpp

```diff
@@ -149,4 +151,40 @@ void* ArrayBufferObject::operator new(size_t size)
     return CustomAllocator<ArrayBufferObject>().allocate(1);
 #endif
 }
+
+Value ArrayBufferObject::getValueFromBuffer(ExecutionState& state, size_t byteindex, TypedArrayType type, bool isLittleEndian)
+{
+    // If isLittleEndian is not present, set isLittleEndian to either true or false.
```

Comment:
How do we know if `isLittleEndian` present or not here?

Reply:
`isLittleEndian` is an optional parameter and we can set this value through builtin functions like below.
```js
// DataView.prototype.setInt32()
setInt32(byteOffset, value, littleEndian) // littleEndian parameter represents the endian option 
```

Reply:
For other cases, we consider only the little endian platform by default.

Reply:
Thanks for the detailed reply.